challenges/classes.js

```diff
@@ -4,4 +4,66 @@
 // console.log(cuboid.volume()); // 100
 // console.log(cuboid.surfaceArea()); // 130
 
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+// function CuboidMaker(cuboidAtributes) {
+// 	this.length = cuboidAtributes.length;
+// 	this.width = cuboidAtributes.width;
+// 	this.height = cuboidAtributes.height;
+// }
+
+// CuboidMaker.prototype.volume = function() {
+// 	return this.length * this.width * this.height;
+// };
+
+// CuboidMaker.prototype.surfaceArea = function() {
+// 	return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+// };
+
+//====================Refactor==================
+
+class CuboidMarker {
```

Comment:
There is a typo here that will throw errors. Should be `Maker` instead of Marker.